gpu/utils/CMakeLists.txt

```diff
@@ -26,5 +26,6 @@ if(build)
     PCL_MAKE_PKGCONFIG("${LIB_NAME}" "${SUBSYS_NAME}" "${SUBSYS_DESC}" "${SUBSYS_DEPS}" "${EXT_DEPS}" "" "" "")
 
     # Install include files
-    PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_PATH}" ${dev_incs} ${incs} ${srcs})
+    PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_PATH}" ${incs} ${srcs})
```

Comment:
Passing `${srcs}` in here doesn't make much sense. Supposedly everything which is in `src/` is not to be distributed/installed, only used for compiling pcl. Does not including it, prevent you from compiling the module?

Reply:
Not including it won't prevent me from compiling the module. I can't figure out why the ${srcs} should be installed either.